Userland/DevTools/UserspaceEmulator/SoftCPU.cpp

```diff
@@ -1871,7 +1882,23 @@ void SoftCPU::FCOMI(const X86::Instruction& insn)
     m_flags_tainted = false;
 }
 
-void SoftCPU::FSTP_RM80(const X86::Instruction&) { TODO_INSN(); }
+void SoftCPU::FSTP_RM80(const X86::Instruction& insn)
+{
+    if (insn.modrm().is_register()) {
+        fpu_set(insn.modrm().register_index(), fpu_pop());
+    } else {
+        // FIXME: Respect shadow values
+        // long doubles can be up to 128 bits wide in memory for reasons (alignment) and only uses 80 bits of precision
+        // gcc uses 12 byte in 32 bit and 16 byte in 64 bit mode
+        // so in the 32 bit case we have to read first, to not override data on the overly big write
+        u128 f80();
```

Comment:
This is a very strange declaration. please use brace initialization instead, i.e `u128 f80 {};`

Reply:
I thought i changed that, but that may explain the build error